### PR TITLE
Updated for store and forward

### PR DESCRIPTION
**Pull request by Jasonvdb**
## Description

- Updated callbacks and FFI for new store and forward
- Cancelling pending transactions after 3 days

## Types of changes

* [ ] UI fix (non-breaking change which fixes a UI issue)
* [ ] Functionality bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [x] New Tests
* [ ] Documentation

## Checklist:

* [ ] I have tested this on multiple simulator devices with different screensizes.
* [x] I'm merging against the `development` branch.
* [x] Commits have been squashed into one commit with a descriptive commit message
* [x] I ran all tests before pushing.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added/changed tests to cover my changes if not UI changes.

### CODE REVIEW

File: MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift [hunk 1]
```diff
@@ -107,8 +106,16 @@ struct TariSettings {
     //Used for showing a little extra detail in the UI to help debugging
     let isDebug = true
     let maxMbLogsStorage: UInt64 = 5000 //5GB
+    let pushNotificationServer = "https://1c1c2bea.ngrok.io"
+    let expirePendingTransactionsAfter: TimeInterval = 60 //A minute
     #else
     let isDebug = false
     let maxMbLogsStorage: UInt64 = 500 //500MB
+    let pushNotificationServer = "https://push.tari.com"
+    let expirePendingTransactionsAfter: TimeInterval = 60 * 60 * 24 * 3 //3 days
```
Comment (kukabi) [inline MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift:115]: I prefer suffixing var names like this with the time unit abbrev (secs, ms, etc), so I know later when I'm reading if it was seconds, minutes, etc.

File: MobileWallet/TariLib/wallet.h [hunk 2]
```diff
@@ -424,6 +426,9 @@ bool wallet_test_mine_transaction(struct TariWallet *wallet, unsigned long long
 // Simulates a TariPendingInboundtransaction being received
 bool wallet_test_receive_transaction(struct TariWallet *wallet,int* error_out);
```
Comment (kukabi) [inline MobileWallet/TariLib/wallet.h:427]: Wrong Swift 😋

File: MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift (hunk 1 above)
Comment (Jasonvdb) [reply MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift:115]: At first I had it as hours and an Int but then I figured if the type was TimeInterval it's implied the var is in seconds?

Comment (kukabi) [reply MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift:115]: I don't know if my point is valid but, imagine I'm reading it textually as I do here without the help of Xcode to see the type.

Comment (kukabi) [reply MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift:115]: It's a minor thing nevermind.

**Pull request merged**